bin/server.js

```diff
@@ -1,5 +1,7 @@
 var express = require('express');
+var cors = require('cors');
 var app = express();
+app.use(cors());
```

Comment:
You can see which middlewares are being applied just looking to that block of code. 

Please note that middleware order does matter.